src/table/utils/handle-key-press.js

```diff
@@ -142,6 +142,15 @@ export const bodyHandleKeyPress = ({
   switch (evt.key) {
     case 'ArrowUp':
     case 'ArrowDown':
+      // Shift + up/down arrow keys: select multiple values
+      // When at the first row of the cell, shift + arrow up key, no value is selected
+      evt.shiftKey &&
+        ((evt.key === 'ArrowUp' && cell.rowIdx !== 0) || evt.key === 'ArrowDown') &&
```

Comment:
what about last row?

Reply:
When at the last row of the cell, there is no specific thing to consider to doing multiple value selections

Reply:
no, when at the last row of the cell, you press shift + arrow down, it just stops there.

Reply:
I agree. Though the engine does not complain, it is better that we avoid undefined: someRowIdx.